tutorials/npm-Enterprise-on-a-closed-network.md

```diff
@@ -0,0 +1,62 @@
+# Npm Enterprise on closed network
+
+You can use npm Enterprise on the private/closed network. Here are the two approaches:
+
+1. Install npmE in open network and move VM’s to closed network
+2. Air-Gap installation
+
+### Install npmE in open network and move VM’s to closed network
+
+To achieve this create two VM’s let’s say npmBlank and npmSource on open-network and install npm Enterprise on both these VM’s.
+
+Both these VM’s originate as npm Enterprise servers, provisioned using the instructions provided [up and running section]( https://npme.npmjs.com/docs/up-and-running/platforms/other.html)
+
+Now, follow the instructions below.
+
+* Seed the VM npmSource (currently connected to the Internet) with the packages you need.
+* For seeding packages to this VM you can use White-listing or mirror policy.
+* Bring this VM (npmSource) to the closed network.
+* Also bring npmBlank that has no packages on it to the closed network. (It’s just a blank installation of npm Enterprise).
+* Now set up the blank npm Enterprise (i.e npmBlank) to mirror the npmSource VM with your packages on it.
+
+Follow the steps below to bring new packages from public Internet to your closed network.
```

Comment:
Please change this block to:

To add new packages from the public registry to your closed network:

 1. Clone `npmeSource` and allow it access to the internet.
 1. Add any new packages to the `whitelist` that were missing from the initial install.
 1. Copy `/usr/local/lib/npme` to a memory stick or portable drive. If you changed the location of the storage paths, be sure to copy the contents from _all_ the folders.
 1. Stop `npmeSource`'s npme instance that's running inside your network from the npme Dashboard.
 1. Copy over all the files from the cloned instance to the correct destination path on `npmeSource`.
 1. Restart the npme instance on `npmeSource` from the npme Dashboard.
 1. Restart replication on `npmePrimary` with the command `npme reset-follower` from the shell.